proto/pom.xml

```diff
@@ -91,6 +109,7 @@
                 <pathconvert pathsep=" " property="proto.files" refid="proto.path" />
                 <exec executable="protoc" failonerror="true">
                   <arg value="--java_out=src/main/java" />
+                  <arg value="--proto_path=/usr/include" />

```

Comment:
I'm not sure if this is the best way to set this value. I was looking for something that would point the maven-antrun-plugin to /usr/include, but I coudln't find any plugin/project parameter that would allow me to do so.

This probably needs to be updated to a value that depends on the OS. What's the best way to do that?